aws/data_source_aws_elb_hosted_zone_id.go

```diff
@@ -50,9 +65,18 @@ func dataSourceAwsElbHostedZoneIdRead(d *schema.ResourceData, meta interface{})
 	if v, ok := d.GetOk("region"); ok {
 		region = v.(string)
 	}
-
+	var i int
```

Comment:
Maybe rename variable to something more descriptive.
lbTypeIndex ?

Reply:
Thanks for the suggestion, I'll add that now.